README.md

```diff
@@ -96,3 +96,32 @@ Import `connect` function from our `./store` and use it for getting state and di
 
 <button on:click="{increment}">+</button>
 ```
+
+## Usage with [@storeon/router](https://github.com/storeon/router)
+If you want to use the @storeon/svelte with the [@storeon/router](https://github.com/storeon/router) you should import the `router.createRouter` from [@storeon/router](https://github.com/storeon/router) and add this module to `createSvelteStore` instead of `createStore`
+
+#### `store.js`
+```js
+import { createSvelteStore } from "@storeon/svelte";
+import router from '@storeon/router'
+const connect = createSvelteStore([
+  router.createRouter([
```

Comment:
it's may not matter, but better to follow one style guide, for example
```js
import { createSvelteStore } from "@storeon/svelte";
import { createRouter } from "@storeon/router";

const connect = createSvelteStore([
createRouter([
...
```

Reply:
Done